src/platforms/stm32/src/lib/gpio_driver.c

```diff
@@ -532,6 +541,10 @@ static Context *gpio_driver_create_port(GlobalContext *global, term opts)
     Context *ctx = context_new(global);
 
     struct GPIOData *gpio_data = malloc(sizeof(struct GPIOData));
+    if (UNLIKELY(IS_NULL_PTR(gpio_data))) {
```

Comment:
I believe the `UNLIKELY` is redundant here

Reply:
I had a mental laps there ;)
Fixed.